crates/accelerate/src/euler_one_qubit_decomposer.rs

```diff
@@ -651,7 +717,7 @@ fn compute_error(
             let num_gates = gates.len();
             let gate_fidelities: f64 = gates
                 .iter()
-                .map(|x| 1. - err_map.error_map[qubit].get(&x.0).unwrap_or(&0.))
+                .map(|x| compute_error_term(x.0.as_str(), err_map, qubit))
```

Comment:
```suggestion
                .map(|gate| compute_error_term(gate.0.as_str(), err_map, qubit))
```
No change with `rustfmt`